less/mixins.less

```diff
@@ -1 +1,7 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.displayFlex(@justify, @align) {
+  display: flex;
+  justify-content: @justify;
+  align-items: @align;
+}
```

Comment:
Just a note: I saw you styled the buttons in your home-page.less file,however the README asked for a parametric mixin for the button.